CODEOWNERS

```diff
@@ -0,0 +1 @@
+*.js    @hdamico
```

Comment:
You can leave just `@hdamico` (to be a global code owner) or change this line for now to `*.rb`. WDYT?